code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -237,7 +239,7 @@
 
 /obj/item/reagent_containers/glass/beaker/drugs
 	name = "baggie"
-	desc = "A baggie. Can hold up to 10 units."
+	desc = "A baggie; I wonder what is in here."
```

Comment:
I think having description like this could be better, since this baggie is only found in the drug dealer ruin
```suggestion
	desc = "A small plastic baggie, often used by pharmaceutical \"entrepreneurs\"."
```